core/templates/components/forms/custom-forms-directives/object-editor.directive.ts

```diff
@@ -68,7 +68,6 @@ angular.module('oppia').directive('objectEditor', [
           'list-of-sets-of-translatable-html-content-ids',
           'list-of-tabs',
           'list-of-unicode-string',
-          'logic-error-category',
           'logic-question',
```

Comment:
Should this be removed?

Reply:
removed

Reply:
I have removed the tempate for logic-error, so I think it makes sense to remove it from here too.
What do you think @vojtechjelinek

Reply:
Agreed.

Reply:
removed 'logic-question' too